packages/documentation-site/patternfly-docs/content/layouts/compass/cards/RecentActivityCard.tsx

```diff
@@ -0,0 +1,171 @@
+    {
+      id: 1,
+      name: 'frontend-deployment',
+      project: 'automation-platform',
+      progress: ['success', 'success', 'success'],
+    },
+    {
+      id: 2,
+      name: 'backend-deployment',
+      project: 'automation-platform',
+      progress: ['success', 'pending', 'default'],
+    },
+    {
+      id: 3,
+      name: 'database-deployment',
+      project: 'automation-platform',
+      progress: ['success', 'success', 'danger'],
+    },
+    {
+      id: 4,
+      name: 'frontend-deployment',
+      project: 'ci-dashboard',
+      progress: ['success', 'warning', 'pending'],
+    },
+  ];
+
+  const iconMap = {
+    success: <CheckCircleIcon />,
+    info: <ResourcesFullIcon />,
+    pending: <Spinner isInline />,
+    danger: <ExclamationCircleIcon />,
+    warning: <ExclamationTriangleIcon />,
+  };
+
+  return (
+    <CompassPanel hasNoPadding isFullHeight>
+      <Card isPlain isFullHeight>
+        <CardHeader
+          actions={{
+            actions: (
+              <Dropdown
+                isOpen={false}
+                onSelect={() => {}}
+                onOpenChange={() => {}}
+                toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+                  <MenuToggle
+                    ref={toggleRef}
+                    variant="plain"
+                    onClick={() => {}}
+                    isExpanded={false}
+                    aria-label="Recent activity card options"
+                  >
+                    <EllipsisVIcon />
+                  </MenuToggle>
+                )}
+                popperProps={{ position: 'right' }}
+              ></Dropdown>
+            ),
+            hasNoOffset: false,
+            className: '',
+          }}
+        >
+          <CardTitle>
+            <Flex alignItems={{ default: 'alignItemsCenter' }}>
+              <PortIcon />
+              <span>Recent activity</span>
+            </Flex>
+          </CardTitle>
+        </CardHeader>
+        <CardBody>
+          <Table aria-label="Recent activity table" variant="compact" className="pf-m-plain">
+            <Thead>
+              <Tr>
+                <Th width={30}>Name</Th>
+                <Th width={30}>Project</Th>
+                <Th width={30}>Progress</Th>
+                <Th width={10} />
+              </Tr>
+            </Thead>
+            <Tbody>
+              {activityData.map((activity, rowIndex) => (
+                <Tr key={activity.id}>
+                  <Td>
+                    <Button variant="link" isInline component="a" href="#">
+                      {activity.name}
+                    </Button>
+                  </Td>
+                  <Td>
+                    <Button variant="link" isInline component="a" href="#">
+                      {activity.project}
+                    </Button>
+                  </Td>
+                  <Td>
+                    <ProgressStepper isCompact aria-label={`Progress for ${activity.name}`}>
```

Comment:
Because two activities have the same name, wind up with the same value in the rotor for two different items. Same with one of the buttons.